templates/hooks/db_instance/sdk_update_post_build_request.go.tpl

```diff
@@ -0,0 +1,3 @@
+	if !delta.DifferentAt("Spec.DBSubnetGroupName") {
```

Comment:
Please put a code comment to indicate why we're doing this :) I will undoubtedly forget in the future :)